### PR TITLE
refactor(rxStatusTags): FRMW-385-convert to Quark Service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-385

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `rxApp.rxStatusTags` to Quark Service.

### CODE REVIEW

File: src/quarks/rxStatusTags/docs/rxStatusTags.docs.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>
+  Primarily used for applications to specify custom status tags.
```
Comment (CITguy) [inline src/quarks/rxStatusTags/docs/rxStatusTags.docs.html]: Please use the description as seen in the ngdocs for the service.

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
## `rxStatusTags` Quark Service

### Demo Page
screen shot 2016-02-05 at 11 26 34 am


### API Docs
screen shot 2016-02-05 at 11 28 49 am

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**